index.html

```diff
@@ -139,7 +139,7 @@
 			}
 
 			function Foo (props, state, context) {
-				return h('div', h('button', {onClick: handleClick, ref: 'button'}, 'Click Me!'), {...props, ...state, ...context})
+				// return h('div', h('button', {onClick: handleClick, ref: 'button'}, 'Click Me!'), {...props, ...state, ...context})
```

Comment:
is the commenting of this file intentional?

Reply:
Yes, was trying to test something on safari and it doesn't support the `{...props}` syntax. I probably should remove this file since it's just a playground to bounce tests for implementation ideas into.